pkg/boot/universalpayload/utilities.go

```diff
@@ -174,6 +174,52 @@ type FbFixScreenInfo struct {
 	Reserved     [2]uint16
 }
 
+type MCFGBaseAddressAllocation struct {
+	BaseAddr  EFIPhysicalAddress
+	PciSegGrp uint16
+	StartBus  uint8
+	EndBus    uint8
+	Reserved  uint32
+}
+
+type ResourceInfo struct {
+	Type        string
+	BaseAddress string
+	EndAddress  string
+	Attributes  string
+}
+
+type ResourceRegions struct {
+	Mmio64Base  uint64
+	Mmio64Limit uint64
+	Mmio32Base  uint64
+	Mmio32Limit uint64
+	IoPortBase  uint64
+	IoPortLimit uint64
+}
+
+const (
+	AcpiMcfgSysFilePath                 = "/sys/firmware/acpi/tables/MCFG"
```

Comment:
probably ACPIMcfgSysFilePath, but here I'm a bit confused with initialism here regarding to Mcfg.